src/content/docs/en/reference/experimental-flags/csp.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: Experimental Content Security Policy
+sidebar:
+  label: Content Security Policy
+i18nReady: true
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 6
+---
+
+import Since from '~/components/Since.astro'
+
+<p>
+
+  **Type:** `boolean | Csp`<br />
+  **Default:** `false`<br />
+  <Since v="5.9.0" />
+</p>
+
+Enables support for [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/CSP) to help minimize certain types of security threats by controlling which resources a document is allowed to load. This provides additional protection against [cross-site scripting (XSS)](https://developer.mozilla.org/en-US/docs/Glossary/Cross-site_scripting) attacks.
+
+Enabling this feature adds additional security to Astro's handling of processed and bundled scripts and styles by default, and is further configurable for more control over these, and additional, content types.
+
+To enable this feature, add the experimental flag in your Astro config:
+
+```js title="astro.config.mjs" ins={4-6}
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+    experimental: {
+      csp: true
+    }
+});
+```
+
+When enabled, Astro will add a `<meta>` element inside the `<head>` element of each page.
+
+This element will have the `http-equiv="content-security-policy"` attribute, and the `content` attribute will provide values for the `script-src` and `style-src` [directives](#directives) based on the script and styles used in the page.
+
+```html
+<head>
+    <meta 
+        http-equiv="content-security-policy" 
+        content="
+            script-src 'self' 'sha256-somehash'; 
+            style-src:'self' 'sha256-somehash'"
+   >
+</head>
+```
+
+## Configuration
+
+You can further customize the `<meta>` tag by enabling this feature with a configuration object that includes additional options.
```

Comment:
I would like to keep the naming consistent. Before we said "element", now we say "tag". I don't have strong opinions about which ones, as long we're consistent

Reply:
```suggestion
You can further customize the `<meta>` element by enabling this feature with a configuration object that includes additional options.
```

Reply:
Changing to `<tag>` since it's only used once here, and `<element>` is used everywhere else!